e2e/packages/loadable/src/dynamic.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Dynamic2 from "./dynamic-2";
+import Static from "./static";
+
+const Dynamic: React.FC = () => {
+  const [toggle, setToggle] = React.useState(false);
+  return (
+    <>
+      <div data-test-id="dynamic-div">I am the Dynamic1 component</div>
+      <button data-test-id="toggle-button" onClick={() => setToggle(!toggle)}>
+        Toggle
+      </button>
+      <div data-test-id="toggle-div">{toggle ? "ON" : "OFF"}</div>
+      <Dynamic2 />
```

Comment:
I guess there is a reason for adding `<Dynamic2 />` here but why isn't there any e2e test in `loadable.spec.js` looking for that component?

Reply:
It's not actually. Fixed and added test for Dynamic2!